src/app.js

```diff
@@ -1,21 +1,47 @@
 'use strict';
+// - You can find a files to test this task in ./public folder.
+// |-- Its already tested with .html\.txt\.css files.
+// |-- Try link like this: http://localhost:/file/index.html =)
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
-
-module.exports = sum;
+const PORT = process.env.PORT || 8000;
+
+const http = require('http');
+const fs = require('fs');
+const path = require('path');
+
+// Server init:
+const server = http.createServer((request, response) => {
+  // Get file name from url params:
+  const fileFromParam = request.url.split('/').slice(1);
+
+  // Hint if the pathname does not start with /file/:
+  if (fileFromParam[0] !== 'file') {
+    response.end(`
+      Please, use address like /file/*filename* to get the file.
+    `);
+
+    return;
+  }
+
+  // Get absolute path to file using 'path' module:
+  const filePath = path
+    .join(__dirname, '/public/' + fileFromParam.slice(1).join('/'));
+
+  // Set response type to text:
+  response.setHeader('Content-Type', 'text/plain');
+
+  try {
+    // Read file and return:
+    const file = fs.readFileSync(filePath, 'utf-8');
+
+    response.end(file);
+  } catch (e) {
```

Comment:
```suggestion
  } catch {
```
Unused var

Reply:
not sure about it) IDE and ESlint would tell me if this var is unused )

![image](https://user-images.githubusercontent.com/89999954/201727792-603fcf5e-6182-481f-a5e1-9614f4e52c6e.png)
![image](https://user-images.githubusercontent.com/89999954/201727894-84ce359a-44ce-4166-a659-5e49dc304191.png)

Reply:
@MaxManis It seems that linter in node projects works with older version of ES (it's strange): https://2ality.com/2017/08/optional-catch-binding.html
I prefer to skip unused error variable in catch block: it's possible by standard.
Leave as is now, but skip where possible.